docs/installation/hardware.rst

```diff
@@ -0,0 +1,118 @@
+
+1. Average number of requests per functional scenario: **7** (higher means more
+   requests per second)
+2. Average waiting time between functional scenario's: **1 minute** (lower means more
+   requests per second)
+
+==================  ============================
+Concurrent users    Expected requests per second
+------------------  ----------------------------
+  100               15
+  250               30
+  500               60
+1.000               120
+2.000               235
+==================  ============================
+
+The above number for requests per second depends greatly on the the actual usage of the
+API. We used theoretical scenarios to give some indication, so it's best to use a
+higher number when looking for the minimum system requirements below.
+
+Minimum system requirements
+---------------------------
+
+* Platform: 64-bit
+* Processor(s): 4 - 16 CPUs (see below) at 2.0 GHz
+* RAM: 8 - 32 GB (see below)
+* Hard disk space: 20 GB (excluding storage for documents)
+
+Based on the number of requests per second you need, you can see what kind of hardware
+you need to achieve this.
+
+======================  ======  ==============
+Requests per second     CPUs    Memory (GB)
+----------------------  ------  --------------
+ 25                      4       8
+ 50                      6      12
+100                     12      24
+150                     14      28
+200                     16      32
+======================  ======  ==============
+
+With these specifications you can run everything on a single machine or divided over
+several instances.
+
+Postgresql database minimum requirements
+----------------------------------------
+
+The performance of Objects API under load is very much dependant on the performance of the Postgresql database. The number of requests per second and the total number of objects may affect the duration of the API calls. In order to avoid performance issues in production and aid in resolving performance issues we recommend to optimize the resources available and fine-tune the Postgresql with the help of `pgbench`. This built-in Postgresql tool gives an indication of the performance of the database setup.
+
+An example run using `pgbench`::
+
+  pgbench (15.14 (Debian 15.14-0+deb12u1))
+  scaling factor: 1
+  query mode: simple
+  number of clients: 16
+  number of threads: 4
+  maximum number of tries: 1
+  duration: 60 s
+  number of transactions actually processed: 65995
+  number of failed transactions: 0 (0.000%)
+  latency average = 14.543 ms
+  latency stddev = 10.885 ms
+  initial connection time = 23.771 ms
+  tps = 1099.724563 (without initial connection time)
+
+Using the default settings of pgbench, the following minimum `tps` is necessary for production setups:
+
+===================  ====
+Max objects per day  tps
+-------------------  ----
+100                200
+1000               500
+10000              1000
+>10000+            1500
+=================  ====
```

Comment:
This table doesn't seem to render currently, might be due to the values in the column not being aligned

<img width="1126" height="445" alt="afbeelding" src="https://github.com/user-attachments/assets/5368555d-cdeb-4485-a5d9-a43e52b5bb0f" />

```suggestion
===================  ====
Max objects per day  tps
-------------------  ----
100                  200
1000                 500
10000                1000
>10000+              1500
===================  ====
```